src/content/docs/reference/astro.mdx

```diff
@@ -255,7 +252,9 @@ and per-page overrides in the
 
 ### Server Endpoints
 
-Arcjet can protect Astro server endpoints (API routes). These run on the server and have access to the request object.
+Arcjet can protect Astro server endpoints (API routes). Keep in mind that Arcjet
```

Comment:
This goes over multiple lines so I can't suggest the change, but I think it could be reworded slightly:

> Arcjet protects dynamic Astro routes, not static routes ([learn more about this](#static-routes)).

Reply:
Oh nice. Didn't know that!